ultra/baselines/ddpg/ddpg/noise.py

```diff
@@ -0,0 +1,46 @@
+"""
+source code: https://github.com/ShangtongZhang/DeepRL/deep_rl/component/random_process.py
```

Comment:
Are the MIT license requirements fulfilled?

Reply:
please ask Nhat, he was fixing the licenses